ban/core/models.py

```diff
@@ -117,9 +118,9 @@ def housenumbers(self):
 
 
 class HouseNumber(Model):
-    # Set of characters from La Poste to optimize OCR removing confusing ones
-    # (like 0/O, 1/I…).
-    CEA_FORMAT = '[\dAB]{2}\d{3}[234679ABCEGHILMNPRSTUVXYZ]{5}'
+    # INSEE + set of characters from La Poste to optimize OCR (removing
```

Comment:
Suggestion:
Set of OCR-friendly characters.
Dropped confusing chars like 0/O, 1/I, ….